.github/workflows/flutter-branch.yml

```diff
@@ -84,57 +80,25 @@ jobs:
           path: ${{ env.CODE_COVERAGE_FILE_PATH }}
           min_coverage: ${{ env.CODE_COVERAGE }}
           exclude: ${{ env.CODE_COVERAGE_EXLUDE_FILE }}
-      
-  android-publish:
-    runs-on: [self-hosted, Linux]
-    needs: [generate-build-number, lint, test]
+
+  build-ios:
+    runs-on: [self-hosted, macOS]
+    needs: [generate-build-number, test, lint]
     steps:
-      - uses: QuickBirdEng/actions/checkout-ssh@main
+      - uses: QuickBirdEng/actions/flutter-build@main
```

Comment:
have you tested the workflow/ the build-ios and build-android isn't going to work if you skip the checkout step. You need to keep the checkout-ssh as it is